src/spicy/spicyz/glue-compiler.cc

```diff
@@ -1194,6 +1194,23 @@ struct VisitorZeekType : hilti::visitor::PreOrder<hilti::Result<hilti::Expressio
         return (! ids.empty() && ids.front().has_value()) ? ids.front()->namespace_() : hilti::ID();
     }
 
+    result_t create_record_type(const hilti::ID& ns, const hilti::ID& local,
+                                const std::vector<hilti::Expression>& fields) {
+        if ( fields.size() ) {
+            SPICY_DEBUG(hilti::util::fmt("Creating Zeek record type %s::%s with fields:", ns, local));
+
+            for ( const auto& f : fields )
+                SPICY_DEBUG(
+                    hilti::util::fmt("  %s",
+                                     f.as<hilti::expression::Ctor>().ctor().as<hilti::ctor::Tuple>().value()[0]));
```

Comment:
Logging the ctr parameters will print e.g., `"foo"` instead of `foo`; we could do some more unwrapping here to get the actual source name.

I think it would also be nice to add `-D zeek` and storing in `output` in the new test so we exercise this logging code.

Reply:
> Logging the ctr parameters will print e.g., `"foo"` instead of `foo`; we could do some more unwrapping here to get the actual source name.

+1

> I think it would also be nice to add `-D zeek` and storing in `output` in the new test so we exercise this logging code.

I'll skip that since we aren't generally testing debug output elsewhere either.